src/Components/CSharpExtensionSupport.fs

```diff
@@ -0,0 +1,51 @@
+namespace Ionide.VSCode.FSharp
+
+open Fable.Import.VSCode.Vscode
+
+module CSharpExtension =
+
+    let private msCSharpExtensionName = "ms-vscode.csharp"
+    let private openvsixCSharpExtensionName = "ms-vscode.csharp"
```

Comment:
Where do these names come from? Are they different than the ids on the store pages?